### DISCUSSION TITLE
Headless with fuzzing

### DISCUSSION THREAD

**Discussion by abdulx01** [projectdiscovery/nuclei-templates#6976] [Ideas]
I am thinking if any possibility for creating a headless template where we can fuzz on URL or raw requests:  This can be for XSS, and other vulnerabilities. The only thing I want to fuzz for DOM XSS, and DOM clobbering where we don't see any reflection to inject payload.
Like one of a jaeles signature about DOM XSS. Likewise, I want to create like this.  If you know kindly help me.

**Comment by olearycrew** [comment]
@ehsandeep do you have an opinion on how we might see combining headless templates with the new fuzzing capabilities? I don't see a reason that wouldn't work today, but you know more about the headless vs. standard engine than I do!

**Reply by ehsandeep** [reply]
We have a open issue for this - https://github.com/projectdiscovery/nuclei/issues/3339
